trashies.go

```diff
@@ -57,9 +57,8 @@ func DeletePerson(w http.ResponseWriter, r *http.Request) {
 }
 
 func launch(w http.ResponseWriter, r *http.Request) {
-	t, _ := template.ParseFiles("index.html")
-	t.Execute(w, nil)
-	log.Println("YEEE")
+	t, _ := template.ParseFiles("index.html", "footer.html", "header.html")
```

Comment:
So just to be clear, we need to do this in every go file we write?

Reply:
we need to do this for every template used to create each page. We can probably make a wrapper function to extract out the repeated stuff.